src/components/RegisterLoginModal/RegisterLoginModal.tsx

```diff
@@ -76,7 +75,7 @@ function RegisterLoginModal() {
             <CloseIcon />
           </IconButton>
           <RegisterLoginHeader checked={checked} onChange={handleChange} onClick={handleClose} state={state} />
```

Comment:
TS will stop complaining if in `RegisterLoginHeader` you rewrite the props to be a single object instead of multiple arguments:
```
interface Props {
  checked: boolean;
  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
  onClick: () => void;
}

function RegisterLoginHeader({ checked, onChange, onClick }: Props): JSX.Element {
```
You also pass a state here but `RegisterLoginHeader` doesn't expect any prop named like this.